pom.xml

```diff
@@ -265,6 +263,44 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>mrm-maven-plugin</artifactId>
+        <version>1.0-beta-2</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>start</goal>
```

Comment:
@stephenc It is probably useless to start/stop mrm outside of the run-its profile ?